docs/solutions/hypervisors/kvm/kvm-baremetal/main.tf

```diff
@@ -0,0 +1,73 @@
+provider "oci" {
+  tenancy_ocid     = "${var.tenancy_ocid}"
+  user_ocid        = "${var.user_ocid}"
+  fingerprint      = "${var.fingerprint}"
+  private_key_path = "${var.ssh_api_private_key_path}"
```

Comment:
This isn't really an "ssh" key... in all other solutions and examples this is called private_key_path and also has a private_key_password, such that the provider looks like:

provider "oci" {
tenancy_ocid = "${var.tenancy_ocid}"
user_ocid = "${var.user_ocid}"
fingerprint = "${var.fingerprint}"
private_key_path = "${var.private_key_path}"
region = "${var.region}"
private_key_password = "${var.private_key_password}"
}